applications/facings-nft-collection-manager-stage-2.md

```diff
@@ -0,0 +1,232 @@
+
+
+### Team Org Repos
+
+- https://github.com/facings
+- https://github.com/FACINGS/eospyo
+
+### Team Member Repos
+
+- https://github.com/robrigo
+- https://drive.google.com/drive/folders/1OfEwjO_ho5QTB9dfrbjMk2IUPEZ6iq6m?usp=sharing
+- https://github.com/roadscape
+- https://github.com/felsin
+- https://github.com/mr-souza
+- https://github.com/pongilo
+- https://github.com/mmoreirasantosdev
+
+
+### Team LinkedIn Profiles (if available)
+
+- https://www.linkedin.com/in/robertkonsdorf/
+- https://www.linkedin.com/in/keirkleinknecht/
+- https://www.linkedin.com/in/mr-souza/
+- https://www.linkedin.com/in/marcos-moreira-51285b16/
+
+
+## Development Status
+
+We recently delivered Stage 1 of this project as an ENF grant and have been preparing for Stage 2 of development, having started  research and preliminary design of the plugin system, and developing numerous UX and usability improvements, as well as progress on TypeScript implementation.
+
+
+## Development Roadmap
+
+
+### Milestone Summary
+
+- **Total Estimated Duration:** 4.5 months
+- **Full-Time Equivalent (FTE):** 4 FTE
+- **Total Costs:** 180,000 USD
+
+
+### Milestone 1 — TypeScript, UX, and Plugins Core
+
+- **Estimated duration:** 2 months
+- **FTE:**  4
+- **Costs:** 80,000 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 0a. | License | GPLv3 |
+| 0b. | Documentation | We will provide an updated ‘Getting Started’ guide and deployment instructions. We will document how to add, configure, and build new plugins. |
+| 0c. | Testing Guide | Documentation for a walkthrough of the UI as described and the intended outcome of each subset will be provided along with expected feedback results. FACINGS UI standards including visibility of system status, user control and freedom, consistency, error prevention, recognition over recall, flexibility and efficiency, minimalistic design, error recovery and documentation will all be implemented. |
+| 0d. | Docker | We will provide a Dockerfile that can be used to launch a UI instance and test all the functionality delivered with this milestone. |
+| 1. | TypeScript Implementation | Converting all components to TypeScript to improve maintainability and reliability. This has been requested by the EOS community. |
+| 2. | UX Improvements | (1) Jungle4 EOS testnet support<br />(2) Airdrop - allow minting to multiple accounts<br />(3) Transfers - filter NFTs by name or collection<br />(4) Upgrade navigation / breadcrumbs and small screen optimization<br />(5) IPFS updates - manual address entry, previews, validation, view originals<br />(6) Metadata handling - new social links for collection, better defaults for schema, allow blank immutable fields, improved input validations |
+| 3. | Plugin Architecture | Design and implement a plugin framework which allows for added functionality at the collection level. |
```

Comment:
The concept of a plugin is vague at this point and I don't know how to technically evaluate it. Will plugins contain only UX code? Some combination of UX and indexing code? Will they have an on-chain component?

Reply:
Our team envision plugins to be mostly UX code, however it will be possible to inject a plugin into the app from an external git repo into the app, and that plugin could have API integrations within it, which could integrate with off-chain indexers and/or the chain RPC, if desired.

We've updated the Plugin Architecture section to be a bit more detailed and hopefully clear up the vagueness. Let me know if that helps!